proto/rill/runtime/v1/resources.proto

```diff
@@ -365,6 +365,9 @@ message AlertSpec {
   int32 intervals_limit = 6;
   bool intervals_check_unclosed = 7;
   uint32 timeout_seconds = 8;
+  // for alerts that have not been edited since resolver and resolver_properties have been added
```

Comment:
When the runtime restarts, it should parse all YAML files and update any resources that output a different proto object. So I don't think this should be necessary? Is the issue perhaps in some other place?

Reply:
Here's the code that triggers a comparison of all resources when the parser first runs: https://github.com/rilldata/rill/blob/main/runtime/reconcilers/project_parser.go#L147

And here's the code that does the comparison for an alert object specifically: https://github.com/rilldata/rill/blob/main/runtime/reconcilers/project_parser.go#L520

Reply:
Good point. There were issues with stage runtimes not coming up. The root cause could have been that. (This PR was raised before it was identified)